### PR TITLE
fix: instance can change class when current and target parent metadat…

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/ADF-92

### CODE REVIEW

File: actions/class.RdfController.php [hunk 1]
```diff
@@ -760,10 +759,9 @@ public function moveInstance()
                         'data'          => $diff
                     ]);
                 }
-
-                $status = $this->getClassService()->changeClass($instance, $destinationClass);
-                $response = ['status'      => $status];
             }
+            $status = $this->getClassService()->changeClass($instance, $destinationClass);
```
Comment (gabrielfs7) [inline actions/class.RdfController.php:763]: We are moving the instance even if confirmed is false if there is no difference

**Review by gabrielfs7 — CHANGES_REQUESTED** [review]
Check the move instance behavior

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: actions/class.RdfController.php [hunk 2]
```diff
@@ -19,6 +19,7 @@
  *               2008-2010 (update and modification) Deutsche Institut für Internationale Pädagogische Forschung (under the project TAO-TRANSFER);
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
  *               2013-2021 (update and modification) Open Assessment Technologies SA;
+ *
```
Comment (gabrielfs7) [inline actions/class.RdfController.php]: why?

**Review by gabrielfs7 — APPROVED** [review]

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-move_resource.playground.kitchen.it.taocloud.org:49021/

**Pull request merged**